python/ray/serve/schema.py

```diff
@@ -538,13 +537,13 @@ class gRPCOptionsSchema(BaseModel):
     )
```

Comment:
If `ServeDeploySchema` is marked as stable, should we also mark `gRPCOptionsSchema` as stable as well? 🤔

Reply:
No it's fine to have sub-fields that can change if they're marked.

gRPC is still experimental, it should be marked alpha